sentinelsat/sentinel.py

```diff
@@ -145,13 +144,14 @@ def query(self, area, initial_date=None, end_date=datetime.now(), **keywords):
         and any other search keywords accepted by the SciHub API.
         """
         query = self.format_query(area, initial_date, end_date, **keywords)
-        self.load_query(query)
-        return self.products
+        return self.load_query(query)
 
-    def load_query(self, query, start_row=0):
+    def load_query(self, query, start_row=0, products=[]):
```

Comment:
I suggest removing the `products=[]` argument altogether. All it is used for is to concatenate new products to an existing list. It makes a lot more sense to do 
```python
products = api.load_query(...)
products += api.load_query(...)
```
outside this method instead.